### PR TITLE
wycheproof and scala dockerfile generation fix

### LINKED ISSUE #3320: Some external projects (wycheproof, scala) Dockefile can't be generated correctly.
Some external projects ( wycheproof) Dockefile can't be generated correctly, some contents are missing.

For example: wycheproof need bazel installed however there is no installation step when build the image by dockerfile.
https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder_Tether/73/console

Believe some information are missing in [set_test_info() ](https://github.com/adoptium/aqa-tests/blob/master/external/common_functions.sh#L195-L213)

A quick look shows that we need to add following test info, similar way as ant_version [ant_version=$(getProperty "ant_version")](https://github.com/adoptium/aqa-tests/blob/master/external/common_functions.sh#L205)

gradle_version
sbt_version
bazel_version
openssl_version

### PR DESCRIPTION
**Pull request by LizyBbethy**
a few updates to the [set_test_info()](https://github.com/adoptium/aqa-tests/blob/master/external/common_functions.sh#L195-L213) function
grinder https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/3496/console
Fixes #3320

### CODE REVIEW

File: external/wycheproof/test.properties [hunk 1]
```diff
@@ -3,5 +3,5 @@ script="wycheproof.sh"
 home_path="\${WORKDIR}"
 test_results="testResults"
 tag_version="master"
-bazel_version="1.2.1"
+bazel_version="4.2.2"
```
Comment (sophia-guo) [inline external/wycheproof/test.properties:6]: I think this change should be part of #3216 to update the bazel version, which should not be part of this PR. This PR is a general fix of parsing and setting set information.

**Comment by sophia-guo** [comment]
Grinder job ` https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/3496/console` failed, which is  bazel specific .

From console output I can see 

```
13:40:15       [exec] Step 9/26 : ARG BAZEL_VERSION=4.2.2
```

which means test information is correctly parsed and set and Dockerfile can be generated correctly. Please remove the changes in test.properties. All others LGTM.

**Review by sophia-guo — APPROVED** [review]

**Review by smlambert — APPROVED** [review]
